### PR TITLE
State file: add canceled DF data transfer page

### PR DESCRIPTION
**Pull request by jenny-heath**
image

image

### CODE REVIEW

**Review by embarnard — APPROVED** [review]
Looks good!

File: app/views/state_file/questions/canceled_data_transfer/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<% title = t(".title_html") %>
+<% content_for :page_title, title %>
+<% content_for :card do %>
+  <h1 class="h2" id="main-question"><%= title %></h1>
+
+  <div class="text--small spacing-below-35">
+    <div class="spacing-above-25"><%= t(".other_options_html") %></div>
+
+    <div class="spacing-above-35"><%= link_to t(".faq"), root_path %></div>
```
[the remark this answers is not in the document]
Comment (jenny-heath) [reply app/views/state_file/questions/canceled_data_transfer/edit.html.erb:9]: yeah it should be more obviously a placeholder

File: app/controllers/concerns/state_file/irs_data_transfer_links_concern.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      link = URI(state_file_fake_direct_file_transfer_page_path)
+      link.query = { redirect: return_url }.to_param
+      link
+    end
+
+    def irs_testing_link
+      if ENV["IRS_TRANSFER_AUTH_URL"].present?
+        irs_testing_link = URI(ENV["IRS_TRANSFER_AUTH_URL"])
+        irs_testing_link.query = { redirect: return_url }.to_param
+      end
```
Comment (bengolder) [inline app/controllers/concerns/state_file/irs_data_transfer_links_concern.rb:15]: Idea: it might be nice to pull the env var into a config variable

**Review by bengolder — APPROVED** [review]
This looks great! Approved!

**Pull request merged**